kraken-std/src/kraken/std/util/check_file_exists_and_is_committed_task.py

```diff
@@ -18,7 +18,7 @@ def to_description(self, file_path: Path) -> str:
         if self == CheckFileExistsAndIsCommittedError.IS_NOT_COMMITTED:
             return f"'{file_path}' exists but is not committed"
 
-        return "IMPOSSIBLE STATE"
+        return "IMPOSSIBLE STATE"  # type: ignore[unreachable]
```

Comment:
Tbh. I don't know why we had it like this here in the first place, imo this should just `assert False`.

```suggestion
        assert False, "unreachable"
```

Reply:
Agreed; can do in a different PR but mypy throws when this isn't here now

Reply:
👍 We pick up the latest Mypy version always so whatever they add in terms of new checks, it may require some code adjustments every now and then.